components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OIDCAuthenticatorConstants.java

```diff
@@ -54,6 +56,9 @@ private OIDCAuthenticatorConstants() {
 
     public static final String SID = "sid";
 
+    public static final Pattern OIDC_BCLOGOUT_ENDPOINT_URL_PATTERN =
+            Pattern.compile("(.*)/identity/oidc/slo?");
```

Comment:
```suggestion
            Pattern.compile("(.*)/identity/oidc/slo(.*)");
```

Reply:
need to have a framework PR for this endpoint as well. please link that

Reply:
fixed in a8d1c44

Reply:
PR for the endpoint - https://github.com/wso2/carbon-identity-framework/pull/3322